### DISCUSSION TITLE
Reason to use your JVM?

### DISCUSSION THREAD

**Discussion by Maelep** [UniversalMediaServer/UniversalMediaServer#2752] [Q&A] [answered]
I see 
```bash
# Use our JVM if it exists
if [ -f jre8/bin/java ]; then
    JAVA="jre8/bin/java"
fi
```

Is there any reason to prefer your JVM than the one installed with my distro?

**Comment by SubJunk** [comment, marked as answer]
In the past we would get a lot of bug reports that ended up being from people using different versions of Java, so that's why we started bundling it. In particular, some of our dependencies (particularly Cling) will only work properly with JRE 8, so when users have JRE 9+ installed, it's better for us to use our one.
